components/automate-ui/src/app/pages/service-groups/service-groups.component.ts

```diff
@@ -221,30 +221,16 @@ export class ServiceGroupsComponent implements OnInit, OnDestroy {
     document.querySelector<HTMLElement>('app-services-sidebar').focus();
   }
 
-  // TODO: @afiune change the backend to send the values already splitted
-  public getPackageNameFromFullRelease(release: string): string {
-    const brokenRel = this.splitFullRelease(release);
-    return brokenRel[0].concat('/', brokenRel[1]);
-  }
-
-  // TODO: @afiune change the backend to send the values already splitted
-  public getShortReleaseFromFullRelease(release: string): string {
-    const brokenRel = this.splitFullRelease(release);
-    return brokenRel[2].concat('/', brokenRel[3]);
-  }
-
-  private splitFullRelease(release: string): string[] {
-    if (release) {
-      const releaseVars = release.split('/');
-      if (releaseVars.length === 4) {
-        return releaseVars;
-      }
-
-      // these returns should never happen, but just to be safe
-      return new Array('-', '-', '-', '-');
+  // TODO @afiune: Add links when they work
+  public tooltipMessageFor(field: string): string {
```

Comment:
uups! I didn't mean to add this code. I was in the middle of two PRs and this code snuck in. Ill delete it.